go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/teamwork/godoc
+
+go 1.12
```

Comment:
You can bump to 1.20 while you're at it 😛

Reply:
Heeeey he's alive! 😁 It's on 1.19 now, you must have seen a older commit 😛